ui/package.json

```diff
@@ -15,12 +15,13 @@
   },
   "devDependencies": {
     "@rollup/plugin-inject": "5.0.5",
+    "@vitejs/plugin-basic-ssl": "^1.1.0",
     "sass-embedded": "1.79.4",
     "typescript": "5.5.3",
     "vite": "5.4.8"
   },
   "dependencies": {
-    "ngeo": "2.9.0-latest.20241030T072255Z.1c68af25a"
+    "ngeo": "2.9.0-latest.20241029T155214Z.b0ee1737f"
```

Comment:
This is a downgrade!

Reply:
was not one this morning.
Thanks

Reply:
Effectively: https://github.com/camptocamp/demo_geomapfish/commit/f1c25b25d18555d5847b8293e8c33342cf844193 :-)